timrlink.net.CLI.Test/ProjectTimeDatabaseExportActionTest.cs

```diff
@@ -0,0 +1,480 @@
+            var projectTimeDatabase2 = memoryContext.ProjectTimes.ToList().First();
+            Assert.AreEqual(true, projectTimeDatabase2.Billable);
+            Assert.AreEqual(true, projectTimeDatabase2.Changed);
+            Assert.AreEqual(false, projectTimeDatabase2.Closed);
+            Assert.AreEqual(12, projectTimeDatabase2.BreakTime);
+            Assert.AreEqual(1000, projectTimeDatabase2.Duration);
+            Assert.IsNotNull(projectTimeDatabase2.Deleted);
+
+            // Here we update the description and return the project time one day earlier
+            projectTime.description = "Dachstein";
+
+            projectTimeServiceMock
+                .Setup(service => service.GetProjectTimes(It.IsAny<DateTime>(), It.IsAny<DateTime>(), null, null,
+                    null, null, null))
+                .ReturnsAsync(projectTimes);
+
+            // Now we return our project time one day earlier and want it to be reactived. Deleted flag must be removed again
+            // We simulate this by returning the project time in this time period
+            var importAction2 = new ProjectTimeDatabaseExportAction(loggerFactory, memoryContext,
+                "2022-10-01", to: "2022-10-02", userServiceMock.Object, taskServiceMock.Object,
+                projectTimeServiceMock.Object);
+            await importAction2.Execute();
+
+            // Now the deleted flag must be removed again
+            Assert.AreEqual(true, projectTimeDatabase.Billable);
+            Assert.AreEqual(true, projectTimeDatabase.Changed);
+            Assert.AreEqual(false, projectTimeDatabase.Closed);
+            Assert.AreEqual(12, projectTimeDatabase.BreakTime);
+            Assert.AreEqual(1000, projectTimeDatabase.Duration);
+            Assert.AreEqual("Dachstein", projectTimeDatabase.Description);
+            Assert.IsNull(projectTimeDatabase.Deleted);
+
+            var metadata = memoryContext.Metadata.FirstOrDefault();
+            Assert.IsNull(metadata);
+        }
+        
+        [Test]
+        public async System.Threading.Tasks.Task TestIfTimezoneAreWorkingCorrectly()
+        {
+            var loggerFactory = LoggerFactory.Create(builder => builder.AddConsole().SetMinimumLevel(LogLevel.Debug));
+
+            const string userUuid = "32c8c87e-43ea-11ed-b878-0242ac120002";
+            const string taskUuid = "2909B8F0-4996-4D51-A2BA-1EB690AB2102";
+            
+            var users = new List<User>
+            {
+                new User 
+                { 
+                    externalId = "John Carmack", 
+                    uuid = userUuid
+                }
+            };
+
+            var task = new Task
+            {
+                name = "Customer B",
+                uuid = taskUuid
+            };
+            var tasks = new List<Task> { task };
+
+            var projectTime = new API.ProjectTime();
+            projectTime.startTime = new DateTime(2022, 10, 2, 10, 30, 0);
```

Comment:
this combination doesn't look like an edgecase at all🤔 

10+7 will almost all the time be the same day (until timezone -3)